src/Body/PlainText.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+namespace Http\Client\Body;
+
+use Http\Client\Body;
+
+/**
+ * Plain text body
+ *
+ * @author Márk Sági-Kazár <mark.sagikazar@gmail.com>
+ */
+class PlainText implements Body
+{
+    /**
+     * @var string
+     */
+    protected $string;
+
+    /**
+     * @param string $string
+     */
+    public function __construct($string)
+    {
+        $this->string = $string;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getContentHeaders()
+    {
+        return [
+            'Content-Type' => 'text/plain',
```

Comment:
You are right. This body type was born at the time I wanted to typehint for the body interface in the HttpMethodsClient. Since then I just worked on it, but didn't rethink if it is useful or not.